elytron/src/main/java/org/wildfly/extension/elytron/RealmParser.java

```diff
@@ -108,6 +108,8 @@ class RealmParser {
             .addAttributes(FileSystemRealmDefinition.HASH_CHARSET)
             .addAttributes(FileSystemRealmDefinition.CREDENTIAL_STORE) // new
             .addAttributes(FileSystemRealmDefinition.SECRET_KEY) // new
+            .addAttribute(FileSystemRealmDefinition.KEY_STORE) //new
```

Comment:
This will need to be updated once my subsystem bump is ready. A new `filesystemRealmParser` variable will need to be added.